app/assess/routes.py

```diff
@@ -114,13 +114,21 @@ def display_sidebar(sub_criteria_id, theme_id):
         Theme.from_filtered_dict(theme) for theme in sub_criteria.themes
     ]
 
+    comments = get_comments(application_id=application_id, sub_criteria_id=sub_criteria_id)
+
     # TODO: would render a higher level page with first theme displaying by default # noqa
     return render_template(
-        "sidebar.html",
+        "sub_criteria.html",
         sub_criteria=sub_criteria,
         themes=themes,
         current_theme_id=theme_id,
         on_summary=on_summary,
+        funding_amount_requested=1920,
+        project_name="Project name",
+        fund="My fund",
+        workflow_status="IN_PROGRESS",
```

Comment:
all hardcode values are taken from pr https://github.com/communitiesuk/funding-service-design-assessment/pull/62 for the base template